projects/components/src/viewer/code-viewer/code-viewer.component.ts

```diff
@@ -204,6 +211,22 @@ export class CodeViewerComponent implements AfterViewInit, OnChanges, OnDestroy
     });
   }
 
+  private isLineHighlighted(lineNum: number): boolean {
+    if (typeof this.highlightText === 'string') {
```

Comment:
Could you also do this?
```
const lowercaseCodeText  = this.codeTexts[lineNum].toLowerCase();
```

Reply:
Updated the function signature and passed it from the parent of this function